README.md

```diff
@@ -149,6 +149,9 @@ You will need to modify the Trusted Entities for each of these roles that you cr
     }
 ```
 
+### Account Aliases
+The config files can set accountAliases, a dictionary from short name to account iam arn, `arn:aws:iam::ACCOUNT-ID-WITHOUT-HYPHENS`.  If you run `hologram use key/rolename`, it will expand it out to the full arn.  This config param is supported on both the server(org wide accounts), or client(individual accounts).
+
```

Comment:
can you add an example config.json file? without looking at the code it's hard to figure out.

is it just this?
```json
{
  "accountAliases": {
    "acct1": "arn:aws:iam::12345678"
  }
}
```